src/picker/rofimoji.py

```diff
@@ -67,11 +67,16 @@ def parse_arguments(self) -> argparse.Namespace:
         parser.add_argument(
             '--action',
             '-a',
-            dest='action',
+            dest='actions',
             action='store',
-            choices=[action.value for action in Action],
-            default=Action.TYPE.value,
-            help='How to insert the chosen characters'
+            type=Action,
+            choices=list(Action),
+            default=[Action.TYPE],
+            nargs='*',
+            metavar='ACTION',
+            help='How to insert the chosen characters. More than one action may be specified in '
+                 'a space separated list (e.g. --action type copy). Options: ' +
+                 ', '.join(f'"{a.value}"' for a in Action)
```

Comment:
There are two major things going on in this block.

* The selection is now stored in `actions` rather than `action` and it is a list of `Action` enum objects rather than a single string (representing the `Action` value).
* The usage text seemed quite long because it was printing the whole list of choices multiple times. It was difficult to notice the new `...` indicating you could specify more than one. To avoid this, I set a `metavar` to `ACTION` and just put the choices in the help text. The help text now looks like this:

```text
usage: rofimoji.py [-h] [--version] [--action [ACTION ...]]
                   <... snip ...>

Select, insert or copy Unicode characters using rofi.

options:
  -h, --help            show this help message and exit
  --version             show program's version number and exit
  --action [ACTION ...], -a [ACTION ...]
                        How to insert the chosen characters. More than one action may be specified in a
                        space separated list (e.g. --action type copy). Options: "type", "copy",
                        "clipboard", "unicode", "copy-unicode", "print"
<... snip ...>
```